### ISSUE TITLE
Screens stop displaying when moved by RemIM frames

### ISSUE THREAD

**Issue by TehSomeLuigi** [MightyPirates/OpenComputers#539]
When moving computers and screens with Remain In Motion frame carriages, the screens stop displaying.

Turning the computer off and on again **does not** help the situation.

Breaking and replacing the screen **does** fix the screen until it is moved again.

For reference, I'm using OpenComputers MC1.7.10-1.3.3.54,
and RemainInMotion 2.1.0, and I tested this by moving the structure downwards. I haven't tested other directions, but I'm going to assume the same happens.

Prioritise this issue however you wish, it's not a huge issue because I'm happy to report that the computers keep running when moved by the frames, they don't crash, and servers can still be accessed by Wireless Terminals. For inter-mod frames compatibility, that's not bad at all!

Cheers, now it's time for me to rule the world with my computer-controlled frames contraptions >:D!

**Comment by fnuecke** [comment]
Thanks! Sounds like the displaylists / tile entity renderer gets confused. I've been wanting to have a look into RemIM compatibility over the weekend anyway (including potentially make the controller work with OC).

**Comment by TehSomeLuigi** [comment]
That'd be beautiful indeed :)

**Comment by fnuecke** [comment]
Hmm, so I just tested this, and it actually works fine for me (single player, server, single screen, multi-block screen, computers and servers, all directions, including across chunk borders). Could you post a screenshot of the setup where it didn't render? And maybe check your logs, who knows.

**Comment by TehSomeLuigi** [comment]
In the latest dev version, I'm unable to reproduce this bug. Maybe it was accidentally fixed?

I'm going to presume this is no longer a problem and therefore close this issue.

**Comment by fnuecke** [comment]
Quite possible. If it pops up again, let me know!

**Comment by Sevalecan** [comment]
I'm having an identical sounding issue on version 1.6.0.9 on MC 1.7.10 with Funky Locomotion frames.. Computer persistence seems OK as my program will continue running (which I observed by changing bundled cables based on a timer after the movement occurred in the same program). The display however just freezes to whatever state it was right before the frame movement happened. The program I wrote quits after it runs, and does not remain in a loop. The display as shown is how it remains, however.

[EDIT: This does NOT appear to be an issue when using Remain in Motion frames and carriages, which I decided to switch to to make my project less complicated.]

My server log is here(The issue persists even with logging off the server and back on):
[https://gist.github.com/Sevalecan/1423519dd70afe3d3d0d638cb73491cd]

![image](https://cloud.githubusercontent.com/assets/5784119/20250234/250d19e8-a9da-11e6-8c8c-da2e0b3e8502.png)

![image](https://cloud.githubusercontent.com/assets/5784119/20250239/41b37eb6-a9da-11e6-87d0-e9cceeee2caa.png)

![image](https://cloud.githubusercontent.com/assets/5784119/20250236/326936bc-a9da-11e6-874c-d3ca87c7d4fa.png)
